db/migrate/20200303102215_change_phony_benchmark_version.rb

```diff
@@ -0,0 +1,7 @@
+class ChangePhonyBenchmarkVersion < ActiveRecord::Migration[5.2]
+  # The `.latest` method of ::Xccdf::Benchmark is using Gem::Version, which
+  # expects the version to have the same versioning syntax as a gemspec.
+  def change
+    ::Xccdf::Benchmark.find_by(ref_id: 'phony_ref_id').update(version: '0.0.0')
```

Comment:
This isn't quite what I suggested. `down` should be no-op, don't you think? Also, we should use `update!` here to raise any errors.

Reply:
@akofink Sure, I've updated it to `update!` - I thought this would be a no-op as Rails wouldn't try run this on rollback if it doesn't know how to do it (it skips changes that it doesn't know how to revert for its migration DSL)